### PR TITLE
test pull request, ab glucose fixed

### PR DESCRIPTION
**Pull request by abthep**
Hello, world!

### CODE REVIEW

File: mim/experiments/ab_glucose.py [hunk 1]
```diff
@@ -15,11 +15,14 @@ class ABGlucose(Experiment, Enum):
         description="Log Reg baseline with Keras, using age, "
                     "gender, + 4 blood samples",
         model=ab_nn.ab_simple_lr,
-        model_kwargs={
-            'epochs': 100,
-            'batch_size': 32,
-            # 'learning_rate': 0.01,
```
Comment (Tipulidae) [inline mim/experiments/ab_glucose.py:21]: - [ ] Don't leave commented code like this

File: mim/experiments/ab_glucose.py [hunk 2]
```diff
@@ -15,11 +15,14 @@ class ABGlucose(Experiment, Enum):
...
-            'compile_args': {'optimizer': 'sgd'}
+        building_model_requires_development_data=True,
+        model_kwargs={},
+        epochs=100,
+        batch_size=32,
+        optimizer={
+            'name': tf.keras.optimizers.SGD,
+            'kwargs': {}
+            # 'kwargs': {'learning_rate': 1e-4, 'epsilon': 1e-3, }
```
Comment (Tipulidae) [inline mim/experiments/ab_glucose.py]: - [x] Commented code

File: mim/experiments/experiments.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ class Experiment(NamedTuple):
     cv_kwargs: dict = {}
     scoring: Any = roc_auc_score
     hold_out: Any = ChronologicalSplit
-    hold_out_size: float = 0.25
```
Comment (Tipulidae) [inline mim/experiments/experiments.py]: I wanted hold_out_size to default to 0.25.

File: mim/models/simple_nn.py [hunk 4]
```diff
@@ -103,7 +103,7 @@ def compile(self, **kwargs):
 
 
 class BasicFF(keras.Sequential):
-    def __init__(self):
```
Comment (Tipulidae) [inline mim/models/simple_nn.py]: Don't want input to constructor here

**Review by Tipulidae — CHANGES_REQUESTED** [review]

File: mim/experiments/ab_glucose.py (hunk 2 above)
Comment (abthep) [reply mim/experiments/ab_glucose.py]: ok

**Review by Tipulidae — APPROVED** [review]
Splendid work AB!

**Pull request merged**